src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -593,7 +594,8 @@ function ReportPreview({
 
             {isHoldMenuVisible && !!iouReport && requestType !== undefined && (
                 <ProcessMoneyReportHoldMenu
-                    nonHeldAmount={!hasOnlyHeldExpenses ? nonHeldAmount : undefined}
+                    // We cannot pay partially if nonHeldAmount is negative
```

Comment:
Let's get rid of the comment now since the new variable name makes it more clear.

Though `!hasValidNonHeldAmount` doesn't seem right now, shouldn't we resolve as `nonHeldAmount` if the nonHeldAmount IS valid?